### ISSUE TITLE
Question about this project

### ISSUE THREAD

**Issue by GRardB** [Karthik777/UrbanDiversity#1]
Hey Karthik,

I couldn't figure out a better way to contact you, so hopefully this works!

I'm trying to create a mobile app using Meteor, but I'm having trouble with `Meteor.call` when it's running on my phone. It works perfectly in my browser, but it seems like none of my Meteor methods are getting called on mobile. I was wondering if you ran into this, and if so, how did you get it working?

Thanks!

**Comment by Karthik777** [comment]
hmmm. could I see your source code or a snippet.You normally have issues with the publish and subscribe with auto-publish off. But meteor.call did not cause problems. But, I'll try and create a sample demo and post it here in a couple of days

**Comment by GRardB** [comment]
Hey Karthik,

Thanks for responding! This was actually for a hackathon project which ended yesterday, and I figured it out, so don't worry :smile: 

The issue was that the mobile app wasn't correctly connecting to the localhost server on my computer. I solved it by using ngrok to forward traffic from a real domain to my local machine, and explicitly setting `mobile-server` when running `meteor run android-device`.

Again, thanks so much for offering to help!
